src/content/en/2019/http2.md

```diff
@@ -165,85 +175,96 @@ Merging these two stats together, we can see the percentage of installs, of each
+Figure 12 shows us which assets are most commonly pushed. JavaScript and CSS are the overwhelming majority of pushed items, both by volume and by bytes. After this, there is a ragtag assortment of images, fonts, and data. At the tail end we see around 100 sites pushing video, which may be intentional, or it may be a sign of over-pushing the wrong types of assets!
 
-One concern raised by some is that HTTP/2 implementations have repurposed the preload HTTP link header as a signal to push. One of the most popular uses of the preload resource hint is to inform the browser of late-discovered resources, like fonts and images, that the browser will not see until the CSS has been requested, downloaded and parsed. If these are now pushed based on that header, there was a concern that reusing this may result in a lot of unintended pushes.
+One concern raised by some is that HTTP/2 implementations have repurposed the preload HTTP `link` header as a signal to push. One of the most popular uses of the preload [resource hint](./resource-hints) is to inform the browser of late-discovered resources, like fonts and images, that the browser will not see until the CSS has been requested, downloaded, and parsed. If these are now pushed based on that header, there was a concern that reusing this may result in a lot of unintended pushes.
 
-However, the relatively low usage of fonts and images may mean that risk is not being seen as much as was feared. `<link rel="preload" ... >` tags are often used in the HTML rather than HTTP link headers and the meta tags are not a signal to push. Statistics in the [Resource Hints](https://almanac.httparchive.org/en/2019/resource-hints) chapter show that less than 1% of sites use the preload HTTP link header, and about the same amount use preconnect which has no meaning in HTTP/2, so this would suggest this is not so much of an issue. Though there are a number of fonts and other assets being pushed, which may be a signal of this.
+However, the relatively low usage of fonts and images may mean that risk is not being seen as much as was feared. `<link rel="preload" ... >` tags are often used in the HTML rather than HTTP `link` headers and the meta tags are not a signal to push. Statistics in the [Resource Hints](./resource-hints) chapter show that fewer than 1% of sites use the preload HTTP `link` header, and about the same amount use preconnect which has no meaning in HTTP/2, so this would suggest this is not so much of an issue. Though there are a number of fonts and other assets being pushed, which may be a signal of this.
 
 To get around this unintended push, you can provide the `nopush` attribute in your preload header:
 
-    link: </assets/jquery.js>; rel=preload; as=script; nopush
+```
+link: </assets/jquery.js>; rel=preload; as=script; nopush
```

Comment:
Is this all that's needed here? thought I had to do more as per https://github.com/HTTPArchive/almanac.httparchive.org/issues/237#issuecomment-550476367

Reply:
It _should_ be all that's needed. I can confirm that it doesn't work without the class name. But then, why aren't `code-block` styles the default for _all_ `pre` elements?

Reply:
Missing a comma

https://github.com/HTTPArchive/almanac.httparchive.org/blob/f48f1eb5ab2ae8e2a2c1477d0f286dca5472016b/src/static/css/page.css#L29-L28

Should this:

```
.code-block pre {
  overflow-y: hidden;
  overflow-x: scroll;
}
```

Be this:

```
.code-block, pre {
  overflow-y: hidden;
  overflow-x: scroll;
}
```

Reply:
Yes! Fixed.